Update convert-user-mailbox-to-shared-mailbox.md (#3547)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md

## Convert a user's mailbox in a hybrid environment

> More details about converting an user mailbox to a shared mailbox, into an Exchange Hybrid environment can be found on:

Reviewer: ```suggestion
For details about converting a user mailbox to a shared mailbox in an Exchange hybrid environment, see [Cmdlets to create or modify a remote shared mailbox in an on-premises Exchange environment](https://support.microsoft.com/office/cmdlets-to-create-or-modify-a-remote-shared-mailbox-in-an-on-premises-exchange-environment-9e83fb59-c001-729c-a4c0-b2964c154b49) and [Shared mailboxes are unexpectedly converted to user mailboxes after directory synchronization runs in an Exchange hybrid deployment](https://docs.microsoft.com/exchange/troubleshoot/user-and-shared-mailboxes/shared-mailboxes-unexpectedly-converted-to-user-mailboxes).
```

Author: Thank you!

> https://support.microsoft.com/office/cmdlets-to-create-or-modify-a-remote-shared-mailbox-in-an-on-premises-exchange-environment-9e83fb59-c001-729c-a4c0-b2964c154b49

Reviewer: ```suggestion
```

Author: Thank you!

> https://docs.microsoft.com/exchange/troubleshoot/user-and-shared-mailboxes/shared-mailboxes-unexpectedly-converted-to-user-mailboxes

Reviewer: ```suggestion
```

Author: Thank you!
